pom.xml

```diff
@@ -113,7 +113,7 @@
     <project.scm.repository>alfresco-java-sdk</project.scm.repository>
     <project.year>2021</project.year>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.version>17</java.version>
+    <java.version>21</java.version>
```

Comment:
Please check whether we can update java compile time as well  to 21

Reply:
As discused confirming the changes with java 21 compile time as well